core-web/libs/sdk/client/src/lib/client/content/builders/collection/collection.ts

```diff
@@ -0,0 +1,334 @@
+        return this.#limit * (this.#page - 1);
+    }
+
+    private get url() {
+        return `${this.#serverUrl}${CONTENT_API_URL}`;
+    }
+
+    /**
+     * This method returns the current query built
+     *
+     * @readonly
+     * @private
+     * @memberof CollectionBuilder
+     */
+    private get currentQuery() {
+        return this.#query ?? this.#defaultQuery;
+    }
+
+    /**
+     * Takes a language id and filters the content by that language
+     *
+     *
+     * @param {number} languageId The language id to filter the content by
+     * @return {CollectionBuilder} CollectionBuilder - A CollectionBuilder instance
+     * @memberof CollectionBuilder
+     */
+    language(languageId: number): this {
+        this.#query = this.currentQuery.field('languageId').equals(languageId.toString());
+
+        return this;
+    }
+
+    /**
+     * Takes a boolean representing if the content should be rendered or not
+     *
+     * @param {boolean} render A boolean representing if the content should be rendered or not
+     * @return {CollectionBuilder} CollectionBuilder - A CollectionBuilder instance
+     * @memberof CollectionBuilder
+     */
+    render(render: boolean): this {
+        this.#render = render;
+
+        return this;
+    }
+
+    /**
+     * Takes an array of constrains to sort the content by field an specific order
+     *
+     * @example
+     * ```javascript
+     * // This will sort the content by title in ascending order
+     * // and by modDate in descending order
+     *  const sortBy = [{ field: 'title', order: 'asc' }, { field: 'modDate', order: 'desc' }]
+     *
+     *  client.content.getCollection("Blog").sortBy(sortBy)
+     *```
+     *
+     * @param {SortBy[]} sortBy Array of constrains to sort the content by
+     * @return {CollectionBuilder} CollectionBuilder - A CollectionBuilder instance
+     * @memberof CollectionBuilder
+     */
+    sortBy(sortBy: SortBy[]): this {
+        this.#sortBy = sortBy;
+
+        return this;
+    }
+
+    /**
+     * Takes a number that represents the max amount of content to fetch
+     *
+     * Note: The limit is set to 10 by default
+     *
+     * @param {number} limit The max amount of content to fetch
+     * @return {CollectionBuilder} CollectionBuilder - A CollectionBuilder instance
+     * @memberof CollectionBuilder
+     */
+    limit(limit: number): this {
+        this.#limit = limit;
+
+        return this;
+    }
+
+    /**
+     * Takes a number that represents the page to fetch
+     *
+     * @param {number} page The page to fetch
+     * @return {CollectionBuilder} CollectionBuilder - A CollectionBuilder instance
+     * @memberof CollectionBuilder
+     */
+    page(page: number): this {
+        this.#page = page;
+
+        return this;
+    }
+
+    /**
+     * Takes a string that represents a Lucene Query that is used to filter the content to fetch.
+     *
+     * Note: The string is not validated, so be cautious when using it.
```

Comment:
Find a better jsdoc param for this.

Also as @link for lucene query doc at dotcms.com